test/Sentry.Tests/Protocol/MeasurementTests.Transaction_Serializes_Measurements.verified.txt

```diff
@@ -1,60 +1,59 @@
 ﻿{
+  type: transaction,
+  event_id: Guid_1,
+  platform: csharp,
+  transaction: name,
+  transaction_info: {
+    source: custom
+  },
+  start_timestamp: DateTimeOffset_1,
   contexts: {
     trace: {
-      op: operation,
-      trace_id: Guid_1,
```

Comment:
This one is actually different and not just a change in order. I'm not sure if that's relevant tho.

Reply:
I think it was a bugfix in Verify.  They are indeed different guids.  It doesn't really matter here though - the main thing this one is testing is the measurements below.

Good eye though! 👍